mandatory/2-function-creation.js

```diff
@@ -34,7 +47,17 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  let newArr = [];
+  for (let i = 0; i < arr.length; i++) {
+  let = numChange = Number(arr[i].toFixed(2));  
```

Comment:
Your code looks great and easy to read.
I think you have too many = signs when declaring the numChange variable.
Was it intentional?
If yes, what was the reason?